### PR TITLE
SCRUM-13 Remove - from README

### PR DESCRIPTION
**Pull request by Manikumar1998**
- Removed - from the README title text
- Added a few new lines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
-# Internal-Survey-Tool
+# Internal Survey Tool
+
```
Comment (Manikumar1998) [inline README.md:2]: Why do we have to add a new line?

Comment (Hemanth694) [inline README.md:2]: Why did you add an extra line?

Comment (Manikumar1998) [reply README.md:2]: I prefer this the code looks better.

Comment (Hemanth694) [reply README.md:2]: Got it

Comment (Hemanth694) [inline README.md:2]: test
test
test

Comment (Hemanth694) [inline README.md:2]: test 1 test

**Review by Hemanth694 — APPROVED** [review]
